frontend/sportsmatch-app/src/components/MatchInProgress.tsx

```diff
@@ -0,0 +1,53 @@
+import '../styles/MatchInProgress.css'
+
+interface InProgressProps {
+  event: {
+    id: number
+    maxElo: number
+    minElo: number
+    dateEnd: string
+    dateStart: string
+    location: string
+    title: string
+    sport: string
+    playerOne: string
+    playerTwo?: string
+  }
+}
+
+function InProgress({ event }: InProgressProps) {
+  return (
+    <>
+      <div className="container-sm">
+        <div className="host">
+          <div className="row">
+            <div className="col position-relative">
+              <a
+                href="test/1"
+                className="overlap position-absolute top-0 end-0"
+              >
+                <img src="./assets/Filter.png" alt="settings" />
+              </a>
+              <h1>
+                Matchmaking
+                <br /> in progress
```

Comment:
I would use just one component for both match in progress and upcoming match - they are almost the same, we are running into danger of having them inconsistent after some time if we keep them separate. We can discuss on saturday